<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1516</width>
    <height>671</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Crazyflie Client Multi-Agent</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Pictures/Pictures/crazyflie_icon.png</normaloff>:/Pictures/Pictures/crazyflie_icon.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="leftVerticalLayout" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="topLeftHorizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="middleLeftHorizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="middleLeftMiddleVerticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightVertical" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidgetMain">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>400</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayoutConsoles" columnstretch="0,0,0">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelDebugConsole">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string> Debug Console </string>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonClearDebugConsole">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QTabWidget" name="tabWidgetDebugConsole">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="iconSize">
           <size>
            <width>5</width>
            <height>5</height>
           </size>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="debugTab1">
           <attribute name="title">
            <string>1</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="textBrowserDebug01"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="debugTab2">
           <attribute name="title">
            <string>2</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="textBrowserDebug02"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="debugTab3">
           <attribute name="title">
            <string>3</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="textBrowserDebug03"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="debugTab4">
           <attribute name="title">
            <string>4</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="textBrowserDebug04"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="debugTab5">
           <attribute name="title">
            <string>5</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="textBrowserDebug05"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="debugTab6">
           <attribute name="title">
            <string>6</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="textBrowserDebug06"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="debugTab7">
           <attribute name="title">
            <string>7</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="textBrowserDebug07"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="debugTab8">
           <attribute name="title">
            <string>8</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="textBrowserDebug08"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1516</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuOperation_Mode">
     <property name="title">
      <string>Operation Mode</string>
     </property>
     <addaction name="actionSettingsOpModeDefault"/>
    </widget>
    <addaction name="menuOperation_Mode"/>
    <addaction name="actionSettingsInteractiveMap"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionSimulationToggle"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuSimulation"/>
  </widget>
  <action name="actionSettingsOpModeRRTPlanner">
   <property name="text">
    <string>RRT Path Planner</string>
   </property>
  </action>
  <action name="actionSettingsOpModeCoopNav">
   <property name="text">
    <string>Coop Navigation</string>
   </property>
  </action>
  <action name="actionSettingsOpModeMctsPlanner">
   <property name="text">
    <string>MCTS Planner</string>
   </property>
  </action>
  <action name="actionSettingsOpModeDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionSettingsInteractiveMap">
   <property name="text">
    <string>Interactive Map</string>
   </property>
  </action>
  <action name="actionSettingsOpModeRRTSim">
   <property name="text">
    <string>RRT Simulation</string>
   </property>
  </action>
  <action name="actionSettingsOpModeMCTSSim">
   <property name="text">
    <string>MCTS Simulation</string>
   </property>
  </action>
  <action name="actionSettingsOpModeCentralFilter">
   <property name="text">
    <string>Central Navigation Filter</string>
   </property>
  </action>
  <action name="actionSettingsOpModeMRSim">
   <property name="text">
    <string>MR Simulation</string>
   </property>
  </action>
  <action name="actionSimulation">
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="actionSimulationToggle">
   <property name="text">
    <string>Activate Simulation</string>
   </property>
  </action>
  <action name="actionSettingsOpModeExploration">
   <property name="text">
    <string>Exploration</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
